media-api/app/controllers/collections.txt

```diff
@@ -0,0 +1,51 @@
+cities
+family
+filmandmusic
+lr
+pp
+saturdayreview
+trv
```

Comment:
Not necessarily as important at this stage, but I think we should encourage less cryptic collections if they aren't going to collide with real words during search.